cev_eris.dme

```diff
@@ -2078,7 +2080,7 @@
 #include "code\modules\projectiles\ammunition\magazines.dm"
 #include "code\modules\projectiles\guns\alien.dm"
 #include "code\modules\projectiles\guns\energy.dm"
-#include "code\modules\projectiles\guns\hatton.dm"
+#include "code\modules\projectiles\guns\Hatton.dm"

```

Comment:
?

Reply:
hell if i know, my VS code does this. Let it be i wont rename it back in every PR